micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -55,6 +56,8 @@
 
     protected final CamundaBpmVersion camundaBpmVersion;
 
+    private final BasicJdbcConfiguration basicJdbcConfiguration;
```

Comment:
please change "private" to "protected" so that potential subclasses can access it.

Reply:
Done